src/gitlab-ci.coffee

```diff
@@ -132,6 +135,47 @@ module.exports = (robot) ->
             return
 
         return "#{major}.#{minor}.#{dev}"
+    triggerBuildPost = (res, updateRichmond, quickly, branch, buildTagPrefix) ->
+        triggerURL = 'https://agra.sdelements.com/api/v4/projects/33/trigger/pipeline/'
```

Comment:
That makes sense, I'll make the changes. @mroote Do you mean the Agra hostname, or Richmond?